src/app/templates/browser/filter-bar.tpl

```diff
@@ -19,6 +19,19 @@
             </ul>
         </li>
 
+    <% }if(typeof rating !== 'undefined'){ %>
+        <li class="dropdown filter ratings">
+            <a class="dropdown-toggle" data-toggle="dropdown" href="#">
+                <%= i18n.__("Rating") %>
```

Comment:
Since the filter's label is just 'Rating' (I agree, keep width minimum) and this sorts based on minimum rating, can the values be change to 1+, 2+, 3+, etc? Or are they already like that and I didnt notice?

Reply:
what about changing them to ">1", ">2", etc.?

Reply:
I think 1+ is a bit more clear, + what mostly everyone including YTS is using for same purpose.